addons/account/static/src/js/account_payment_field.js

```diff
@@ -72,7 +72,9 @@ var ShowPaymentLineWidget = AbstractField.extend({
                         ref: content.ref,
                         account_payment_id: content.account_payment_id,
                     }));
-                    $content.filter('.js_unreconcile_payment').on('click', self._onRemoveMoveReconcile.bind(self));
+                    var unreconcile_button = $content.filter('.js_unreconcile_payment')
+                    unreconcile_button.on('click', self._onRemoveMoveReconcile.bind(self));
+
```

Comment:
what's that for?

Reply:
That's an archeological vestige of the previous version of the code. I'm removing it :p